docs/source/developer/node-with-devstack.rst

```diff
@@ -0,0 +1,274 @@
+Deploying Node with DevStack and Ironic
+=======================================
+
+This covers the steps to deploy a node using Ironic and set up DevStack with Ironic integration, including creating fake nodes in Ironic.
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Image That We Used
+------------------
+
+For this, we used the Ubuntu 22.04 image to provision a node. You can use any image that is compatible with DevStack. You can download the image from the official Ubuntu website:
```

Comment:
Can you link to a devstack page that lists compatible images? It probably makes sense to also say that the instructions below are ubuntu-specific.